src/backends/github/implementation.js

```diff
@@ -139,8 +139,10 @@ export default class GitHub {
       let url = `https://raw.githubusercontent.com/${repo}/${this.branch}${path}`;
 
       // Assets uploaded to private repos will need valid access tokens.
-      const isPrivateRepo = await this.api.isPrivateRepo();
-      if (isPrivateRepo) {
+      if (!this.isPrivateRepo) {
```

Comment:
This condition would result in an unauthenticated check on every persist for public repos, maybe we should check for the presence of a Boolean instead.

Reply:
@erquhart How do you suggest checking for a boolean? I'm caching this value in th line below so that it only gets called once per CMS session.

Reply:
Oops, thank you!